test/steps/graphql.go

```diff
@@ -57,3 +58,25 @@ func (data *Data) graphqlRequestOnDataIndexReturnsProcessInstancesProcessNameWit
 		return false, nil
 	})
 }
+
+func (data *Data) graphqlRequestOnDataIndexReturnsJobsIDWithinMinutes(id string, timeoutInMin int) error {
+	serviceName := infrastructure.DefaultDataIndexName
+	query := getJobsIDQuery
+	path := "graphql"
+
+	framework.GetLogger(data.Namespace).Debugf("graphqlRequestOnDataIndexReturnsJobsIDWithinMinutes with service %s, path %s, query %s and timeout %d", serviceName, path, query, timeoutInMin)
+	routeURI, err := framework.WaitAndRetrieveRouteURI(data.Namespace, serviceName)
+	if err != nil {
+		return err
+	}
+	response := GraphqlDataIndexJobsQueryResponse{}
+	return framework.WaitForSuccessfulGraphQLRequest(data.Namespace, routeURI, path, query, timeoutInMin, &response, func(response interface{}) (bool, error) {
+		resp := response.(*GraphqlDataIndexJobsQueryResponse)
+		for _, processInstance := range resp.Jobs {
+			if processInstance.ID == id {
```

Comment:
Isn't this ID of a job in general?

Reply:
this is the job, I did not change the variable name in copy/paste...